### PR TITLE
JP-688 Additions to have MIRI MRS dedicated background targets listed in a science association

### PR DESCRIPTION
**Pull request by ddavis-stsci**
… in a science association

### CODE REVIEW

File: jwst/associations/lib/rules_level3_base.py [hunk 1]
```diff
@@ -840,3 +897,27 @@ def _init_hook(self, item):
...
+
+class AsnMixin_AuxData:
+    """Process special and non-science exposures as science.
+    """
+    def get_exposure_type(self, item, default='science'):
+        """Override to force exposure type to always be science
+        Parameters
+        ----------
+        item : dict
+            The pool entry to determine the exposure type of
```
Comment (hbushouse) [inline jwst/associations/lib/rules_level3_base.py]: Ack! Dangling participle (or something like that). How about "The pool entry for which the exposure type is determined".

Comment (ddavis-stsci) [reply jwst/associations/lib/rules_level3_base.py]: Updated the text. I also missed adding the diff of the truth file and the output file so for completeness,
diffs output_pool/jw00623-o003_spec3_001_asn.json /Users/ddavis/data/JWST/CTeam/associations/jw00623-o003_spec3_001_asn.json
    "asn_rule": "candidate_Asn_Lv3Spec",		      |	    "asn_rule": "candidate_Asn_IFU",
    "code_version": "0.13.2a.dev102+g3f32b46e",		      |	    "code_version": "0.13.2a.dev153",
    "constraints": "DMSAttrConstraint({'name': 'program', 'so |	    "constraints": "DMSAttrConstraint({'name': 'program', 'so
            "name": "jw00623-o003_t002_miri_clear",	      |	            "name": "jw00623-o003_t002_miri",

>
So only the asn rule name, the code version and the list of constraints are different.

**Comment by ddavis-stsci** [comment]
Changed the name of the class (Asn_Lv3Spec) to Asn_Lv3SpecAux

Updated code so that the test_level3_basics passes
pytest tests/test_level3_basics.py
==================================================== test session starts ====================================================
platform darwin -- Python 3.6.8, pytest-4.5.0, py-1.8.0, pluggy-0.11.0
rootdir: /Users/ddavis/src/JWST/scsb/jwst, inifile: setup.cfg
plugins: requests-mock-1.5.2, remotedata-0.3.1, openfiles-0.3.2, doctestplus-0.3.0, arraydiff-0.3, ci-watson-0.3
collected 8 items
tests/test_level3_basics.py ........                                                                                  [100%]
================================================= 8 passed in 2.07 seconds =================================================
by limiting the new test to MIR_MRS exposure types

File: CHANGES.rst [hunk 2]
```diff
@@ -1,4 +1,13 @@
-0.13.3 (Unreleased)
+0.13.4 (Unreleased)
+===================
+
+associations
+---
+
+- MIRI MRS dedicated background exposures are now listed as science observations in
+  a new association. [#3459]
```
Comment (nden) [inline CHANGES.rst]: This needs to be the PR #, i.e. 3542

**Pull request merged**